en/3-transaction/transactions.md

```diff
@@ -0,0 +1,257 @@
+   "type":"ContractTransaction",
+   "version":0,
+   "attributes":[
+
+   ],
+   "vout":[
+      {
+         "n":0,
+         "asset":"0xc56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b",
+         "value":"1000",
+         "address":"Ae2d6qj91YL3LVUMkza7WQsaTYjzjHm4z1"
+      },
+      {
+         "n":1,
+         "asset":"0xc56f33fc6ecfcd0c225c4ab356fee59390af8560be0e930faebe74a6daff7c9b",
+         "value":"99999000",
+         "address":"AWHX6wX5mEJ4Vwg7uBcqESeq3NggtNFhzD"
+      }
+   ],
+   "vin":[
+      {
+         "txid":"0x3631f66024ca6f5b033d7e0809eb993443374830025af904fb51b0334f127cda",
+         "vout":0
+      }
+   ],
+   "sys_fee":"0",
+   "net_fee":"0",
+   "scripts":[
+      {
+         "invocation":"406a159b7552c7eaedc79abc86faeca7aa50af52aaa0f14aa9a4abaf498f270a140709992253df55de1b2fd93a6ea13b5344dacbd4e54e4f661fe073edeb72e2f740e28c0866c2ea963e40f8f6edbc1e40b76181fef43a4016d234602a52b31b83f02d745d57188cd72fcb1a8394a39d77270334374848266bb87a29fa4114d1b13240c1e7eae0e8e8d33b1a16c8ece8e96bc832d8f0a069499b8b9590609d8cd2a799a555f5433bdc153466bf6eefea0a568bd08b28afabfacb673785fe8d59ab82ea404874390b85c4d37d3645e03cae571000f3ca344452c2a4018aab57f73750dfb695c5488e3c9887699a2ff69e539b7e37278f470b03bc357ebaad25c397ef3104",
+         "verification":"542102486fd15702c4490a26703112a5cc1d0923fd697a33406bd5a1c00e0013b09a7021024c7b7fb6c310fccf1ba33b082519d82964ea93868d676662d4a59ad548df0e7d2102aaec38470f6aad0042c6e877cfd8087d2676b0f516fddd362801b9bd3936399e2103b209fd4f53a7170ea4444e0cb0a6bb6a53c2bd016926989cf85f9b0fba17a70c2103b8d9d5771d8f513aa0869b9cc8d50986403b78c6da36890638c3d46a5adce04a2102ca0e27697b9c248f6f16e085fd0061e26f44da85b58ee835c110caa5ec3ba5542102df48f60e8f3e01c48ff40b9b7f1310d7a8b2a193188befe1c2e3df740e89509357ae"
+      }
+   ]
+}
+```
+
+### Type
+There are several different types of transactions. Each type has a different purpose and different characteristics.
+
+- Available transaction types:
+  - `ContractTransaction`
+  - `InvocationTransaction`
+  - `ClaimTransaction`
+  - `MinerTransaction`
+  - `StateTransaction`
+- Deprecated transaction types should not be used anymore:
+  - `RegisterTransaction`
+  - `EnrollmentTransaction`
+  - `IssueTransaction`
+- Removed transaction types:
+  - `PublishTransaction`
+  - `VotingTransaction`
+  - `AgencyTransaction`
+
+### Version
+The version attribute allows updates to the transaction structure with backwards compatibility. Currently version `0` and `1` are supported for `InvocationTransaction` and version `0` is the only supported version for any other transaction type.
+
+### Attributes
+Depending on the transaction type it is possible to add attributes. For each attribute a usage type has to be specified, together with the external data and the size of the external data.
+
+| Field    | Type  | Description                           |
+|----------|-------|---------------------------------------|
+| `usage`  | uint8 | Attribute usage type                  |
```

Comment:
Usage should be enum(byte), data should be byte array, and there's not a variable with the name 'length' existing as defined in Neo.Network.P2P.Payloads.TransactionAttribute in NEO C#.